README.md

```diff
@@ -16,15 +16,23 @@
 
 ## Getting Started
 
-To get a local copy up and running follow these simple example steps:
-clone the repository to your local machine
 
 ### VScode, GIT bash
 
 ### clone the repository to your local machine
+To get a local copy and running follow these simple example steps.
 
-### Install
+- Set up a new repository and prepare it for development using best practices (e.g. linters).
```

Comment:
- [ ] [OPTIONAL] We have to add commands not just instructions, in this case, you can do something like so: `git clone project url` for cloning, how to run the project `npm install` ` npm start`. We should always think about non technical people when we are setting up a readme file.